cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java

```diff
@@ -187,8 +182,19 @@ private String tripleStore(Properties p) {
                                 getFormat(),
                                 p,
                                 PROFILE_USED_FOR_INITIAL_STATE_VALUES_PARAMETER,
-                                defaultValueConfig)
-                );
+                                defaultValueConfig))
+                .setStoreCgmesModelAsNetworkExtension(
+                        ConversionParameters.readBooleanParameter(
+                                getFormat(),
+                                p,
+                                STORE_CGMES_MODEL_AS_NETWORK_EXTENSION_PARAMETER,
+                                defaultValueConfig))
+                .setStoreCgmesTerminalMappingAsNetworkExtension(
```

Comment:
why not store the whole conversion context as a separate network extension? I see that it is good to have separate extensions for CGMES model and for conversion context, but I think we could just save the whole context for debugging/validation purposes, instead of creating one extension for each piece of the context. What do you think?

Reply:
Yes, I agree, I made changes in this direction.